src/sql/base/browser/ui/taskbar/media/actionBar.css

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.monaco-action-bar.vertical .action-item .action-label.separator {
```

Comment:
Why is this necessary? Have you verified that this isn't going to cause any problems in the other places that use actionbar? You may want to consider scoping this down to just execution plan unless you think this is something that affects all action bars.

Reply:
Currently, the separator in vertical action bars do not look as intended. This is a targeted fix for that. There isn't any other side effects I have noticed. 

This is how separators looks without this css.

![image](https://user-images.githubusercontent.com/6816294/191424938-016942ed-b86e-49e9-be8e-85dc17d81842.png)